packages/grid/x-data-grid-pro/src/tests/rowEditing.DataGridPro.test.tsx

```diff
@@ -1121,7 +1121,7 @@ describe('<DataGridPro /> - Row Editing', () => {
         const cell = getCell(0, 2);
         userEvent.mousePress(cell);
         fireEvent.doubleClick(cell);
-        await act(() => {
+        await waitFor(() => {
```

Comment:
I don't understand why, but without this change the test was failing with:
```
The current testing environment is not configured to support act
```

See https://app.circleci.com/pipelines/github/mui/mui-x/30074/workflows/a9f04000-72a1-436d-819b-360e41eabb40/jobs/172385

Reply:
Thanks, Matheus!